src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -214,7 +213,7 @@ private static void AddPopParams(AuthenticationRequestParameters authenticationR
                     IdToken = authResult.RawIdToken,
                     CorrelationId = correlationId,
                     Scope = authResult.GrantedScopes,
-                    ExpiresIn = DateTimeHelpers.GetDurationFromWindowsTimestamp(expiresOn, logger),
+                    ExpiresIn = (long)(authResult.ExpiresOn - DateTime.UtcNow).TotalSeconds,
```

Comment:
`authResult.ExpiresOn.Kind` is `Unspecified`. Wonder we need to explicitly specify Utc kind, otherwise might get some funky datetime business.

Reply:
I propose that we use `DateTimeOffset` instead of `DateTime`. MSAL should not be using `DateTime` in general, because we don't display zone-specific dates.